src/scripts/create_schibsted.py

```diff
@@ -1,12 +1,27 @@
 """Create the Schibsted summarisation dataset."""
 
-import fasttext
 import pandas as pd
 from constants import MAX_NUM_CHARS_IN_ARTICLE, MIN_NUM_CHARS_IN_ARTICLE
 from datasets import Dataset, DatasetDict, Split, load_dataset
-from huggingface_hub import HfApi, hf_hub_download
+from huggingface_hub import HfApi
 from requests import HTTPError
 
+NEWSROOM_TO_LANGUAGE = {
+    "sno-commercial": "no",
+    "vektklubb": "no",
+    "e24": "no",
+    "e24partnerstudio": "no",
+    "dinepenger": "no",
+    "vgpartnerstudio": "no",
+    "bt": "no",
+    "tekno": "no",
+    "vg": "no",
+    "ap": "no",
+    "randaberg24": "no",
+    "ab": "se",
+    "sa": "no",
+}
```

Comment:
`se` is Sweden the country, not Swedish the language - the language is `sv`.

```suggestion
NEWSROOM_TO_LANGUAGE = {
    "sno-commercial": "no",
    "vektklubb": "no",
    "e24": "no",
    "e24partnerstudio": "no",
    "dinepenger": "no",
    "vgpartnerstudio": "no",
    "bt": "no",
    "tekno": "no",
    "vg": "no",
    "ap": "no",
    "randaberg24": "no",
    "ab": "sv",
    "sa": "no",
}
```